counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
```

Comment:
If a reissuance specifies a different subasset in the description, then the original subasset wins - that is the purpose of the reissued_asset_longname variable.  I wrote a test to cover this scenario (https://github.com/deweller/counterparty-lib/blob/cip4-wip-2/counterpartylib/test/fixtures/vectors.py#L1527-L1529).

I can add a sanity check here in case someone accidentally appends the wrong subasset into the description of a reissuance.  But that would make the reissuance transaction invalid.

So - the question is should a reissuance with a valid numeric asset and invalid subasset in the description be valid or invalid?

Scenario:
- I own PARENTASSET.mychild (as A100001)
- I issue 100 additional A100001, but I include `;;lPARENTASSET.oopsie` in the description

Should the reissuance of 100 A100001 be a valid or invalid issuance?  

It is ok to let it be valid.  The code will correctly insert PARENTASSET.mychild into the issuances table as the asset_longname.

I lean toward allowing the transaction and ignoring the bad description.  But I'm fine either way.  I'm not worried about the numeric asset collision.